controller/src/main/java/io/pravega/controller/task/Stream/StreamMetadataTasks.java

```diff
@@ -579,11 +583,16 @@ public void setRequestEventWriter(EventStreamWriter<ControllerEvent> requestEven
                 .collect(Collectors.toList()));
     }
 
-    private CompletableFuture<Void> notifyDeleteSegment(String scope, String stream, int segmentNumber) {
+    public CompletableFuture<Void> notifyDeleteSegment(String scope, String stream, int segmentNumber) {
```

Comment:
This is because this class is now a helper class with helper methods that are called from different Stream workflows (like this one is called from truncate and could have been called from deleteStream)

Reply:
Ok.